airflow/operators/python.py

```diff
@@ -123,6 +123,9 @@ def my_python_callable(**kwargs):
     :param templates_exts: a list of file extensions to resolve while
         processing templated fields, for examples ``['.sql', '.hql']``
     :type templates_exts: list[str]
+    :param show_return_value_in_logs: a bool value whether to show return_value
```

Comment:
I think it would be great to explain when you would like to set it to "false" (i.e. explain why we have  this parameter).

Reply:
I just edited the docstring explanation, but
I will think about it a little more and improve it soon.

Reply:
I think it's good enough. Hapy to merge it @KennethanCeyer if you make it ready for review (It's Draft) currently.